### PR TITLE
Add Quinine container (resolves #146, #149)

### LINKED ISSUE #146: ADAM build fails due to not pushing spark-and-maven container
```
docker build -no-cache -t quay.io/ucsc_cgl/adam-build:c251f79c6bde3dce12e685c6cf03d5b1c30e9273--6f908333f4002bdf047e81f8e1c8c25b083640ea . 
Warning: '-no-cache' is deprecated, it will be replaced by '--no-cache' soon. See usage.
Sending build context to Docker daemon 2.048 kB
Sending build context to Docker daemon 2.048 kB

Step 1 : FROM quay.io/ucsc_cgl/spark-and-maven:1.5.2.3.3.3-build
Pulling repository quay.io/ucsc_cgl/spark-and-maven
Tag 1.5.2.3.3.3-build not found in repository quay.io/ucsc_cgl/spark-and-maven
```

### PR DESCRIPTION
**Pull request by fnothaft**

### CODE REVIEW

**Comment by fnothaft** [comment]
Still testing, so have tagged with `needs work`.

**Comment by fnothaft** [comment]
@hannes-ucsc I've removed the `needs work` tag. Please merge for Cloud Pilot.

**Comment by heuermh** [comment]
LGTM

File: quinine-pipelines/test.py [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+                       os.environ["CROMWELL_HOME"],
+                       jars)
+            return None
+            
+        else:
+            _log.error('Found no jars in $CROMWELL_HOME (%s).', os.environ["CROMWELL_HOME"])
+            
+    else:
+            
+        cromwell_exe = distutils.spawn.find_executable("cromwell")
```
Comment (hannes-ucsc) [inline quinine-pipelines/test.py:43]: The rest of us use `which` from BPL.

File: quinine-pipelines/test.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+        # build commandline
+        tool = ['quay.io/ucsc_cgl/quinine-pipelines']
+        base = ['docker', 'run']
+
+        # Check base call for help menu
+        out = subprocess.check_output(base + tool)
+        self.assertTrue('usage: metrics.py [-h] {rna,targeted,contamination}' in out)
+
+
+    def test_docker_call_targeted(self):
```
Comment (hannes-ucsc) [inline quinine-pipelines/test.py:66]: I'd rather have an explanation what `_targeted` means, …

File: quinine-pipelines/test.py [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+        # get working directory
```
Comment (hannes-ucsc) [inline quinine-pipelines/test.py:68]: … than this obvious comment.

File: quinine/test.py [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+import subprocess
+import tempfile
+import unittest
+
+
+class TestQuinine(unittest.TestCase):
+
+    def test_docker_call(self):
+        out, err = check_docker_output(tool='quay.io/ucsc_cgl/quinine')
+        self.assertTrue('A refreshing treatment for all quality control ailments' in out)
```
Comment (hannes-ucsc) [inline quinine/test.py:12]: :-)

**Pull request merged**